tests/playwright/sanity/modules/playing-cards/widget.test.js

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+import EditorSelectors from '../../../selectors/editor-selectors';
+
+test.describe( 'Playing Cards @playing-cards', () => {
+	test( 'Playing Cards widget', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = await wpAdmin.openNewPage();
+
+		await test.step( 'Add widget', async () => {
+			await editor.addWidget( 'playing-cards' );
+
+			const widget = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.container );
+
+			await expect( await widget.count() ).toBe( 1 );
+		} );
+
+		await test.step( 'Add a card', async () => {
+			const addItemButton = await page.locator( '.elementor-repeater-add' );
+
+			await addItemButton.click();
+
+			await expect( await editor.getPreviewFrame().locator( `${ EditorSelectors.playingCards.container } ${ EditorSelectors.playingCards.card }` ) ).toHaveCount( 1 );
+		} );
+
+		await test.step( 'Update card properties', async () => {
+			const card = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.card );
+			const cardHeader = await editor.getPreviewFrame().locator( EditorSelectors.playingCards.cardHeader );
+
+			// Test Diamonds 3
+			await editor.setSelectControlValue( 'card_type', 'diamond' );
+			await editor.setSelectControlValue( 'card_value', '3' );
+
+			await expect( card ).toHaveCSS( 'color', 'rgb(217, 83, 79)' );
+			await expect( cardHeader ).toHaveText( '3' );
+
+			// Test Hearts 4
+			await editor.setSelectControlValue( 'card_type', 'heart' );
```

Comment:
You extract a function to a helper.js to set card type and value in one step to simplify your test code.  This often makes it easier understand when your PR is reviewed speeding up approval.